frappe/core/doctype/module_def/module_def.py

```diff
@@ -64,6 +66,7 @@ def on_trash(self):
 
 		modules = None
 		if frappe.local.module_app.get(frappe.scrub(self.name)):
+			delete_folder(self.module_name,"Module Def",self.name)
```

Comment:
Can you please format this? You can setup pre-commit to automate this.

https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist#linting

Reply:
Resolved